src/bindings/python/src/pyopenvino/graph/node_input.cpp

```diff
@@ -91,6 +99,12 @@ void regclass_graph_Input(py::module m) {
                 get_rt_info : RTMap
                     A dictionary of user defined data.
              )");
+    input.def("replace_source_output",
+              &ov::Input<ov::Node>::replace_source_output,
+              py::arg("new_source_output"),
+              R"(
+                Replaces the source output of this input.
+              )");
```

Comment:
Missing info about parameter!

Reply:
Added missing info. Could you take a look if it is fine?